src/main/app/case/formatter/address.ts

```diff
@@ -1,37 +1,58 @@
+import { capitalize } from 'lodash';
+
 import { Case } from '../case';
 import { CaseData, YesOrNo } from '../definition';
 
-export const fromApi = (data: Partial<CaseData>): Partial<Case> => {
-  const addressParts = data.derivedPetitionerHomeAddress?.split('\n') || [];
-  if (addressParts.length !== 5) {
+export const fromApi = (data: Partial<CaseData>, address: 'your' | 'their'): Partial<Case> => {
+  const isPetitionerAddress = address === 'your';
+  const apiAddress = isPetitionerAddress ? data.derivedPetitionerHomeAddress : data.derivedRespondentHomeAddress;
+  const apiIsAddressInternational = isPetitionerAddress
+    ? data.petitionerHomeAddressIsInternational
+    : data.respondentHomeAddressIsInternational;
+  if (apiIsAddressInternational === YesOrNo.YES) {
```

Comment:
does this not need an else to set to NO?

Reply:
👍 Good idea, I've made this clearer in f881f03.